source/core/views.txt

```diff
@@ -14,9 +14,26 @@ Views
    :depth: 1
    :class: singlecol
 
+MongoDB supports creating read-only views from existing collections
+or other views. For example, you can:
 
-Starting in version 3.4, MongoDB adds support for creating read-only
-views from existing collections or other views.
+- Create a view that :pipeline:`filters <$match>` only low-quantity
+  items from a collection of inventory data.
```

Comment:
I don't quite know why you would need a view for this.  Could you explain a bit regarding your thinking?

Reply:
Discussed Offline, removing.